src/components/PersonalDetails/PersonalDetails.js

```diff
@@ -0,0 +1,200 @@
+                type="Number"
+                style={{
+                  width: 350,
+                  border: "0.5px solid",
+                  color: "#FF5555",
+                }}
+                className="shadow rounded"
+              />
+            </div>
+          </div>
+
+          <div className="mt-5">
+            <h3 style={{ color: "#FF5555" }}>Address</h3>
+            <div className="row d-flex justify-content-around mt-4 ">
+              <div>
+                <label>City</label>
+                <br />
+                <input
+                  placeholder="City Name"
+                  type="text"
+                  style={{
+                    width: 350,
+                    border: "0.5px solid",
+                    color: "#FF5555",
+                  }}
+                  className="shadow rounded"
+                />
+              </div>
+              <div>
+                <label>Street</label>
+                <br />
+                <input
+                  placeholder="Street"
+                  type="text"
+                  style={{
+                    width: 350,
+                    border: "0.5px solid",
+                    color: "#FF5555",
+                  }}
+                  className="shadow rounded"
+                />
+              </div>
+            </div>
+            <div className="row d-flex justify-content-around mt-4 ">
+              <div>
+                <label>State</label>
+                <br />
+                <input
+                  placeholder="State Name"
+                  type="text"
+                  style={{
+                    width: 350,
+                    border: "0.5px solid",
+                    color: "#FF5555",
+                  }}
+                  className="shadow rounded"
+                />
+              </div>
+              <div>
+                <label>Zip Code</label>
+                <br />
+                <input
+                  placeholder="Zip Code"
+                  type="text"
+                  style={{
+                    width: 350,
+                    border: "0.5px solid",
+                    color: "#FF5555",
+                  }}
+                  className="shadow rounded"
+                />
+              </div>
+            </div>
+            <div className="row d-flex mt-4 justify-content-center">
+              <input
+                className="m-2 "
+                type="submit"
+                value="Update"
+                style={{
+                  background: "#00008B",
+                  color: "white",
+                  width: 200,
+                  height: 40,
+                  borderRadius: "40px",
+                }}
+              />
+              <input
+                className="m-2 "
+                type="submit"
+                value="Cancel"
+                style={{
+                  background: "gray",
+                  color: "white",
```

Comment:
this is copy pasted from the previous input adding a className and adding the css there would be a better solution